lib/js/index.js

```diff
@@ -171,6 +180,13 @@ class DateTimePicker extends Events {
     return this;
   }
 
+  clickClockMinutes(e) {
```

Comment:
(Github won't let me add a comment to the right line) - do you need to change `clickClock` to `clickClickHour`?

I changed `clickClock` recently to call `this.set` (to fix #110) - can you update this method to follow the same pattern?